.storybook/stories/combobox/combobox.stories.ts

```diff
@@ -34,8 +35,6 @@ const defaultStory: Story = args => ({
         </clr-options>
       </clr-combobox>
     </clr-combobox-container>
-
-    <img style="display: none;" [src]="'https://raw.githubusercontent.com/vmware-clarity/ng-clarity/main/logo.png?' + controlDisabled" (load)="controlDisabled ? formControl.disable() : formControl.enable()">
```

Comment:
This was a hack to solve to a change detection issues when the disable arg is changed in the Storybook UI. Now, the disabled state doesn't change unless you click in the preview frame after changing the arg.

Reply:
I do not experience any problems with this story. It may be something specific for the Reactive forms, while I changed it to Template Driven.
My guess is that templates ensure a bit more aggressive change detection, which helps here.